app/build.gradle

```diff
@@ -68,6 +68,7 @@ dependencies {
     implementation 'androidx.vectordrawable:vectordrawable:1.1.0'
     implementation 'androidx.preference:preference:1.2.0'
 
+    implementation 'com.github.quickpermissions:quickpermissions-kotlin:0.4.0'
```

Comment:
The library hasn't received an update since the end of 2019 :/

Reply:
Don't worry, I've forked it and updated everything:
https://github.com/cyb3rko/QuickPermissions-Kotlin

Reply:
I've looked through the repo, nicely done!